### PR TITLE
CB-12813 ios Fix target chooser to use full target name

### PR DESCRIPTION
**Pull request by tsubik**
### Platforms affected

iOS

### What does this PR do?

Fixes target Chooser to use full target name with OS version instead of only device name.

### What testing has been done on this change?

I'm using cordova-paramedic for testing my own plugin. Integration testing with choosing the right target.

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: lib/ParamedicTargetChooser.js [hunk 1]
```diff
@@ -128,12 +128,10 @@ ParamedicTargetChooser.prototype.chooseTargetForWindows = function (emulator, ta
 
 ParamedicTargetChooser.prototype.chooseTargetForIOS = function (emulator, target) {
     logger.info('cordova-paramedic: Choosing Target for iOS');
-    var simulatorModelId = util.getSimulatorModelId(this.cli, target);
-    var split            = simulatorModelId.split(', ');
-    var device           = split[0].trim();
-    var simId            = util.getSimulatorId(simulatorModelId);
+    var deviceOSVersion = util.getSimulatorModelId(this.cli, target);
```
Comment (janpio) [inline lib/ParamedicTargetChooser.js]: variable name is strange.

**Comment by janpio** [comment]
Fixed a merge conflict.

Change itself looks good and solves the problem of the JIRA issue, but I am not sure if this could have negative effects on how we currently use paramedic.

**Comment by janpio** [comment]
Issue link was https://issues.apache.org/jira/projects/CB/issues/CB-12813

**Comment by janpio** [comment]
@erisu Is this already implemented this way now after your last changes?

**Comment by erisu** [comment]
@janpio yes current master is already implemented with these changes. I will close this PR.

tsubik, thank you for the contribution.

**Pull request closed without merging**